offchain/metadata/models/token.py

```diff
@@ -30,3 +32,23 @@ def validate_chain_identifier(cls, chain_identifier):
             )
 
         return chain_identifier
+
+    # There are cases where unicode characters (\xf0\x9f\x99\x82\\n\\) can appear in strings
+    # (ex: Edu Nouns) blindly using unicode_escape will cause the string to be escaped incorrectly
+    # only escape strings that can't be parsed as json.
+
+    # Some uri strings are escaped incorrectly because of unicode_escape being used to deal with
+    # unicode characters in the uri. This validator deals with those incorrectly escaped strings
+    @validator("uri")
+    def validate_token_uri(cls, uri):
```

Comment:
@lara-jeftic could you write a test for this?

Reply:
@erikreppel test added